back/app/models/app_configuration.rb

```diff
@@ -4,22 +4,28 @@
 #
 # Table name: app_configurations
 #
-#  id         :uuid             not null, primary key
-#  name       :string
-#  host       :string
-#  logo       :string
-#  header_bg  :string
-#  favicon    :string
-#  settings   :jsonb
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
-#  style      :jsonb
+#  id                     :uuid             not null, primary key
+#  name                   :string
+#  host                   :string
+#  logo                   :string
+#  header_bg              :string
+#  favicon                :string
+#  settings               :jsonb
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#  style                  :jsonb
+#  homepage_info_multiloc :jsonb
```

Comment:
Why isn't it just another field in settings json?

Reply:
The main reason is because we want to support text images. Text images currently only work on multiloc attributes and cannot be nested deeper in a JSON attribute.